src/Paket.Core/InstallProcess.fs

```diff
@@ -94,9 +85,39 @@ let processContentFiles root project (usedPackages:Map<_,_>) gitRemoteItems opti
             |> Seq.toList
 
         let copyContentFiles (project : ProjectFile, packagesWithContent) =
+            let contentFileBlackList : list<(FileInfo -> bool)> = [
+                fun f -> f.Name = "_._"
+                fun f -> f.Name.EndsWith ".transform"
+                fun f -> f.Name.EndsWith ".tt"
+                fun f -> f.Name.EndsWith ".ttinclude"
+                fun f -> f.Name.EndsWith ".install.xdt"
+                fun f -> f.Name.EndsWith ".uninstall.xdt" ]
```

Comment:
couldn't this be achieved with an active pattern for `EndsWith` and a pattern match on `fileinfo.Name`?